gio/src/dbus_node_info.rs

```diff
@@ -0,0 +1,38 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use std::ffi::CStr;
+
+use glib::translate::ToGlibPtr;
+
+use crate::{DBusInterfaceInfo, DBusNodeInfo};
+
+impl DBusNodeInfo {
+    pub fn path(self) -> &'static str {
```

Comment:
```suggestion
    pub fn path<'a>(&'a self) -> &'a str {
```

probably. Can this be `NULL`, if so it should probably be

```suggestion
    pub fn path<'a>(&'a self) -> Option<&'a str> {
```

Reply:
Done